backend/fs/DimensionFS.cpp

```diff
@@ -425,6 +425,22 @@ std::vector<double> RangeDimensionFS::ticks() const {
     return ticks;
 }
 
+std::vector<double> RangeDimensionFS::ticks(size_t start, size_t count) const {
+    std::vector<double> ticks;
+    /*
+    Group g = redirectGroup();
+    if (g.hasData("ticks")) {
+        g.getData("ticks", ticks);
+        return ticks;
+    } else if (g.hasData("data")) {
+        g.getData("data", ticks);
+        return ticks;
+    } else {
+        throw MissingAttr("ticks");
+    }
```

Comment:
Just kill the dead code?

Reply:
well, it is not actually dead, it would be needed, if we had that dataset stuff implemented in the FS backend. I can remove it if you insist

Reply:
I did not know that you were planning to have a `DataSet` abstraction in the FS backend. ... maybe our front-end/back-end separation is at the wrong level then... oh well. No no, I don't insist .. the FIXME could use some explanation though ;)

Reply:
fixmes got more text, not sure, when we will ever resolve them...